Makefile

```diff
@@ -771,6 +771,11 @@ apackage: platform/android/gradle/configuration.gradle
 android-ui-test: platform/android/gradle/configuration.gradle
 	cd platform/android && $(MBGL_ANDROID_GRADLE) -Pmapbox.abis=all :MapboxGLAndroidSDKTestApp:assembleDebug :MapboxGLAndroidSDKTestApp:assembleAndroidTest
 
+#Run instrumentations tests on MicroSoft App Center
+.PHONY: android-test-app-center
+android-test-app-center:
```

Comment:
can `android-test-app-center` be renamed to `run-android-test-app-center` ? This would match the other commands more, non-run commands are used for building.

Reply:
ok